client/db/bolt/db.go

```diff
@@ -49,6 +51,8 @@ var (
 	byteFalse      = encode.ByteFalse
 	byteEpoch      = uint16Bytes(uint16(order.OrderStatusEpoch))
 	byteBooked     = uint16Bytes(uint16(order.OrderStatusBooked))
+	backupDir      = "backup"
+	backupFile     = "backup.kv"
```

Comment:
Backup works.  I recovered from backup.kv with no trouble.  However, the backup should have a .db extension to match the live db extension.  Ideally even the same file name, just in the backup folder.

Reply:
resolved.